### ISSUE TITLE
Allow the user to chose the providers order

### ISSUE THREAD

**Issue by Evpok** [atom/autocomplete-plus#772]
While developing [latex-autocomplete](https://github.com/Evpok/latex-autocomplete) I run into an issue : the suggestions from [latex-completions](https://github.com/JunoLab/atom-latex-completions) are overriding mine, as they set their `inclusionPriority` to 5 and mine is 1. Now, if I want my completions to get at the top, i could simply set it to 6, but then if JunoLabs wanted their completions at the top they'd set it to 7 and so on.

Long story short: all users might not want their suggestions in the same order, and it would be nice if there was a user-level way to chose the provider order.
